bot.py

```diff
@@ -2137,6 +2165,90 @@ async def on_guild_channel_update(
                 f"Category {before.id} was updated but is not associated with a role in cache. This could be an error."
             )
 
+@bot.event
+async def on_guild_emojis_update(guild: discord.Guild, before: Sequence[discord.Emoji], after: Sequence[discord.Emoji]):
+    bot_commands = bot.get_channel(BOT_COMMANDS_ID)
+
+    # Determine if change was made in control server
+    changed_in_hub = guild.id == HUB_SERVER_ID
+
+    # Determine operation and execute
+
+    # Add
+    if len(before) < len(after):
+        emoji = discord.utils.find(lambda e: e not in before, after)
+        if not emoji:
+            Log.error('find() returned None on detected added emoji')
+            return
+        
+        # Automatically sync throughout all guilds if made in control
+        if changed_in_hub:
+            await sync_add(bot=bot, emoji=emoji)
+            
+        # Send View and wait for acceptance or denial
+        else:
+            # TODO: Format based on desired goals
+            await bot_commands.send(
+                f'An emoji, {emoji.name} with the appearence {emoji} has been added to Guild {guild.name} Would you like to sync this change?', 
```

Comment:
If an emoji is uploaded directly to the hub server, it should not send a request. It should merely say "an emoji with appearance (appearance) has been distributed to all servers."

Reply:
currently it syncs the emoji instantly but still sends the approval prompt, however all the buttons fail upon interaction.

Reply:
Should be a simple check like

```python
if guild.id == HUB_SERVER_ID:
    # No request, just sync
```

Reply:
I believe I have something like that. I have a variable called `changed_in_hub` and if 
```python 
if changed_in_hub:
            await sync_add(bot=bot, emoji=emoji)
```
which is then followed by the else block meaning that the view is only sent if the add was not done in the hub, so shouldn't that have the desired functionality?

Reply:
Is there maybe another place in the function or the `sync_` functions that sends a sync request that slipped through?